include/miral/miral/window_management_options.h

```diff
@@ -53,7 +53,7 @@ inline auto add_window_manager_policy(std::string const& name, Args&... args) ->
 class WindowManagerOptions
 {
 public:
-    WindowManagerOptions() = default;
+    WindowManagerOptions();
```

Comment:
This looks like the wrong fix. I suggest:
```c++
    WindowManagerOptions() = delete;
```
as that does not change the meaning of the code.

Reply:
Hah! The warning proves valuable already. I thought the intent of `WindowManagerOptions() = default` was to make a no-parameter constructor available :grin:

Reply:
That is a natural reading. But as clang told you that's not what it does, and if you look at the class semantics it doesn't make sense either.